include/seqan3/range/views/drop.hpp

```diff
@@ -112,64 +117,11 @@ namespace seqan3::views
- * | std::ranges::bidirectional_range |                                    | *preserved*                            |
- * | std::ranges::random_access_range |                                    | *preserved*                            |
- * | std::ranges::contiguous_range    |                                    | *preserved*                            |
- * |                                  |                                    |                                        |
- * | std::ranges::viewable_range      | *required*                         | *guaranteed*                           |
- * | std::ranges::view                |                                    | *guaranteed*                           |
- * | std::ranges::sized_range         |                                    | *preserved*                            |
- * | std::ranges::common_range        |                                    | *preserved*                            |
- * | std::ranges::output_range        |                                    | *preserved*                            |
- * | seqan3::const_iterable_range     |                                    | *preserved*                            |
- * |                                  |                                    |                                        |
- * | std::ranges::range_reference_t   |                                    | std::ranges::range_reference_t<urng_t> |
- *
- * See the \link views views submodule documentation \endlink for detailed descriptions of the view properties.
- *
- * ### Return type
- *
- * | `urng_t` (underlying range type)                                                              | `rrng_t` (returned range type)  |
- * |:---------------------------------------------------------------------------------------------:|:-------------------------------:|
- * | `std::basic_string const &` *or* `std::basic_string_view`                                     | `std::basic_string_view`        |
- * | `std::ranges::borrowed_range && std::ranges::sized_range && std::ranges::contiguous_range`    | `std::span`                     |
- * | `std::ranges::borrowed_range && std::ranges::sized_range && std::ranges::random_access_range` | `std::ranges::subrange`         |
- * | *else*                                                                                        | *implementation defined type*   |
- *
- * The adaptor is different from std::views::drop in that it performs type erasure for some underlying ranges.
- * It returns exactly the type specified above.
- *
- * ### Complexity
- *
- * Construction time of the returned view is in \f$ O(1) \f$ if the underlying range models at least
- * std::ranges::random_access_range and std::ranges::sized_range; otherwise in \f$ O(drop\_size) \f$.
- *
- * ### Example
- *
- * \include test/snippet/range/views/drop.cpp
- *
- * \hideinitializer
+/*!\deprecated Use std::views::drop or seqan3::views::type_reduce | std::views::drop.
```

Comment:
Shouldn't we keep the docs until it is actually removed?

Reply:
Yeah, we could. What do you prefer?

Reply:
Would prefer keeping it, but it's not that incredibly important since we will remove it soon anyways